devnotes/payload_starting.md

```diff
@@ -191,6 +194,28 @@ pflash -P PAYLOAD -r /tmp/skiboot.bin
 tail -c +$((0x1001)) skiboot.bin | unxz > skiboot.lid
 ```
 
+### Building skiboot.lid
+
+   1. Clone the skiboot repository
+   ```
+   git clone https://git.raptorcs.com/git/talos-skiboot
+   ```
+
+   2. Start docker container
+   ```
+   docker run --rm -it -v $PWD:/home/skiboot/skiboot -w /home/skiboot/skiboot coreboot/coreboot-sdk:65718760fa /bin/bash
+   ```
+
+   3. Build the skiboot image
+   It is required, to override `CWARNS` variable and disable treating warnings
+   as errors, or otherwise, skiboot won't build.
+   ```
+   make -j`nproc` CROSS=powerpc64-linux-gnu- CWARNS=-w
```

Comment:
@IgorBagnucki What kind of warnings we face? Can we dump a log maybe? I'm not sure if this would be fine for the upstream.

Reply:
During testing I forgot to checkout correct revision. After checkout, there are no warnings, and overwriting `CWARNS` is not required.